src/components/Main/Schedule/Home.tsx

```diff
@@ -1,51 +1,87 @@
 import React, { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { useQuery } from 'react-query';
+import { useQuery, useMutation, useQueryCache } from 'react-query';
 
 import { Scheduler } from './Scheduler';
 import { ScheduleTabs } from './ScheduleTabs';
+import { NewScheduleForm } from './NewScheduleForm';
 
 import { logout } from '../../../store/actions';
 import { useSelector } from '../../../shared/utilities';
 import { getScheduleData } from '../../../query/schedules';
+import { addSchedule } from '../../../store/apis/schedules';
 
+import { Dialog } from '@material-ui/core';
 import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
 import { buttonTheme } from '../../../shared/styles/theme.js';
 import { useSpinner } from '../../../shared/styles/loading-spinner';
 
+import AddIcon from '@material-ui/icons/Add';
+
 export const Home = () => {
   const classes = useStyles();
-  useSpinner();
+  // useSpinner();
```

Comment:
@CedThen already showed me a problem with useSpinner not playing nice with react query useMutation. Looks like we'll need a more involved solution for the spinner. I'll look into maybe an interface/wrapper to make it easy to specify when to start/stop upon API completion.

Reply:
@CedThen - I got a fix for the spinner [here](https://github.com/House-on-a-Rock/tjc-scheduling-web/pull/36/files#diff-7b4f7e8958ec472ac00c2ace4e37d1e0f6b4508eb484c2d006e73be0e52a3695)